config/routes.rb

```diff
@@ -8,7 +8,9 @@
     }
     resources :discounts
     resources :tables
-    resources :catalogs
+    resources :catalogs do
+      resources :posts, except: :index, shallow: true
```

Comment:
shallow: true lam gi the e

Reply:
để bỏ đi /catalog/:catalog_id trong URI Pattern của posts (ngoại trừ :new, :edit)

Reply:
ready